src/braket/aws/aws_device.py

```diff
@@ -78,11 +79,19 @@ class AwsDevice(Device):
         "Xy": "XY",
     }
 
-    def __init__(self, arn: str, aws_session: Optional[AwsSession] = None):
+    def __init__(
+        self,
+        arn: str,
+        aws_session: Optional[AwsSession] = None,
+        noise_model: Optional[NoiseModel] = None,
```

Comment:
* How does this extend to the emulating the device? If we wanted to pass in a set of device capabilities, would that be a separate attribute?
* Did we consider alternatives like having a method instead (something like `.assume_device_profile()`), having a separate class etc?

Reply:
I view the noise model as the interface for the device capabilities/properties. There are two user stories: (1) I want to emulate a device noise, and (2) I want to study how algorithm result decays with noise. For US#2 it's easier to use noise model directly. US#1 can be solved by implement a `NoiseModel.assume_device` method.

Reply:
Agree with Tim; the way I see it, both are good to have: a way to easily emulate a device, as well as to attach an arbitrary noise model to emulate a hypothetical device.